.eslintrc.js

```diff
@@ -30,5 +30,7 @@ module.exports = {
     ],
     "capitalized-comments": "off",
     "max-params": "off",
+    "jsdoc/newline-after-description": "off",
+    "jsdoc/tag-lines": "off",
```

Comment:
Are the lint deprecated warnings from Estate fixed in this version? If not, we should fix them.